templates/includes/_login.html.twig

```diff
@@ -5,27 +5,37 @@
 
 <!-- Modal -->
 <div class="modal fade" id="loginModal" tabindex="-1" aria-labelledby="loginModalLabel" aria-hidden="true">
-    <div class="modal-dialog">
-        <div class="modal-content">
-            <div class="modal-header">
-                <h5 class="modal-title" id="exampleModalLabel">Se connecter</h5>
-                <button type="button" class="btn-close" data-bs-dismiss="modal" aria-label="Close"></button>
-            </div>
-            <form action="{{ path('app_login') }}" method="post">
+    <form id="formLogin" action="{{ path('app_login') }}" method="post">
+        <div class="modal-dialog">
+            <div class="modal-content">
+                {% if error and error != 'login' %}
```

Comment:
can error have "login"? => if so, it's not really intuitive

Reply:
ok done